pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -250,6 +252,26 @@ public class ServerCnx extends PulsarHandler implements TransportCnx {
 
     private final long connectionLivenessCheckTimeoutMillis;
 
+    private static final LongAdder totalPendingBytes = new LongAdder();
+    private static volatile Gauge pendingPublishBufferUsage;
```

Comment:
It's not a great idea to add static fields. This should be handled in the BrokerService, I believe.